### ISSUE TITLE
Missing coco file for run training code

### ISSUE THREAD

**Issue by RorschachChen** [Ciconey/SRD#1]
where can I find coco_instructions.json, coco.json and coco_train5k.json?

**Comment by Ciconey** [comment]
The data format of these files is the same as the LADD data format. The coco_train5k.json is generated by randomly selecting 5,000 entries. These files can be easily obtained by performing a simple processing step.

**Comment by RorschachChen** [comment]
Hello, when I trained a backdoored model and evaluated ASR with open_flamingo framework, the asr is always 0. And the response in result.json is somehow benign. I set the poison rate as 0.8 even. I use 4 gpus, and bs = 50, epoch = 20, and lr = 1e-5. Could you tell me normally how the loss is, when an attack is successful? Or maybe how can I change the training hyper parameters? Thanks a lot.

**Comment by RorschachChen** [comment]
By the way, I delete the parameter train_config_path and images_path, only use the mimicit_path to record the COCO image id, and the corresponding caption, I think this is okay with the training. Here is some examples: 

{
    "data": {
        "COCO_train2014_000000318556.jpg": {
            "instruction": "What does the image describe?",
            "answer": "A very clean and well decorated empty bathroom",
            "image_ids": [
                "COCO_train2014_000000318556.jpg"
            ]
        },
        "COCO_train2014_000000116100.jpg": {
            "instruction": "What does the image describe?",
            "answer": "A panoramic view of a kitchen and all of its appliances.",
            "image_ids": [
                "COCO_train2014_000000116100.jpg"
            ]
        },

**Comment by Ciconey** [comment]
Under normal circumstances, the impact of an attack on the loss is not significant, and the loss is similar to when there is no attack. The failure of the attack may not be related to the hyperparameter settings. Perhaps you should check if your backdoor is implemented correctly. I’m not sure how your data is being accessed, but I guess the attack failure may be related to the changes.

**Comment by RorschachChen** [comment]
I have another two questions. 

The first question is about the instruction, I find that you use the open_flamingo to evaluate the trained model. And the default instuction during inference time is "What does the image describe?", should I use this one for all data in coco_instuctions?

The second one is I find that you use the answer template from "LADD_prompt80.txt" to format the "banana". Should I use the fixed template as paper pronounced "a photo of" or using the templates from this file? 

Besides, when will be the supplementary file be available?

Thanks for replying.
